doc/source/news.rst

```diff
@@ -19,15 +19,16 @@ Improvements
 
 * [:doc:`/install/ubuntu`] Added support for MariaDB 10.6 on Ubuntu 22.04 (Jammy).
 
-* Added support for aborting query with timeout.[GitHub #344][Reported by Kazuhiko]
+* Added support for execution timeout parameter.[GitHub #344][Reported by Kazuhiko]
   
-  MySQL uses ``max_execution_time`` and MariaDB uses ``max_statement_time`` for aborting query with timeout.
+  MySQL/MariaDB can abort a query if the execution timeout parameter is specified and a execution time exceeds 
+  a time specified with the parameter.
+  The execution timeout parameter is ``max_execution_time`` in MySQL and ``max_statement_time`` in MariaDB.
```

Comment:
We should use upper case into `max_execution_time`. 
Because `MAX_EXECUTION_TIME(N)` is official name.

See: https://dev.mysql.com/doc/refman/5.7/en/optimizer-hints.html#optimizer-hints-execution-time

Reply:
The lower case used in server-system-variables.

https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_max_execution_time

I think it is better to use the lower case because we cannot use Optimizer-Hint (`/*+ MAX_EXECUTION_TIME(N) */`)

Reply:
I misunderstood about https://github.com/mroonga/mroonga/pull/558#discussion_r1006598702 .
I thought `MAX_EXECUTION_TIME(N)` don't work and we must use `max_execution_time` server-system-variables.

Reply:
Fixed to use `MAX_EXECUTION_TIME`